akka-http-core/src/main/scala/akka/http/impl/engine/client/PoolConductor.scala

```diff
@@ -63,17 +67,18 @@ private object PoolConductor {
                                   +---------+
 
   */
-  def apply(slotCount: Int, pipeliningLimit: Int, log: LoggingAdapter): Graph[Ports, Any] =
+  def apply(slotSettings: PoolSlotsSetting, pipeliningLimit: Int, log: LoggingAdapter): Graph[Ports, Any] =
```

Comment:
Do we vote in this case or what? :) I don't have any strong opinions here. 
I would prefer having this additional safety net, but I understand that it's probably the only purpose of `PoolSlotsSetting`